src/dswx_sar/metadata.py

```diff
@@ -301,12 +301,16 @@ def compute_layover_shadow_coverage(data_array, spatial_coverage):
     """
     layover_shadow_pixels = np.sum(
         data_array == band_assign_value_dict['layover_shadow_mask'])
+    total_pixels = data_array.size
+    invalid_pixels = np.sum(
+        data_array == band_assign_value_dict['no_data'])
+    valid_pixels = total_pixels - invalid_pixels
 
-    if spatial_coverage > 0:
+    if valid_pixels > 0:
         return round(layover_shadow_pixels /
-                     (spatial_coverage * data_array.size), 4)
+                     valid_pixels * 100, 4)
     else:
-        return np.nan
+        return 0
```

Comment:
```suggestion
        return 0.0
```
just a nit to make sure to return floating point number.